modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/Modal.js

```diff
@@ -418,10 +422,11 @@ class Iframe extends React.Component {
 				className={classNames({
 					hide: this.state.loading,
 				})}
+				id={this.state.id}
```

Comment:
Yeah, but there's a difference between "not adding an `_iframe_` suffix and not allowing an `iframeId`. 

The latter is a likely needed API.

Reply:
Would there be value in adding `iframeId` prop, that would directly set it? Is the autogenerated `${id}_iframe_` enough? Is there a need to autogenerate it always? We could also add generic [`iframeProps` prop](https://github.com/liferay/clay/blob/44ca5b5ca269f84fc42c78af0ca35cb4fb977a1d/packages/clay-modal/src/Body.tsx#L28) like they did in Clay, `id` could be added through it.

Reply:
Autogenerated doesn't make sense because it doesn't discriminate when it is needed or not. As you can guess, it's not always needed and the fact that you don't know when it is, is why an API is needed.

> We could also add generic iframeProps prop like they did in Clay, id could be added through it.

Sounds reasonable